internal/consensus/reactor.go

```diff
@@ -826,66 +844,65 @@ OUTER_LOOP:
 		}
 
 		time.Sleep(conR.conS.config.PeerGossipSleepDuration)
-		continue OUTER_LOOP
 	}
 }
 
-func (*Reactor) gossipVotesForHeight(
+func pickVotesForHeight(
```

Comment:
Is this name precise? We don't pass a height to this method. It Would it be the current node's height?

Reply:
Also, it returns a single vote, not multiple.

Reply:
We started calling it "bla" (no joke!), so I think it improved :-D

Reply:
About the height, we're using the heights inside the structs we are passing as params

Reply:
About `votes`, I think you got a point. Let me rename

Reply:
As @cason is 20 cm away from me, we agreed on this name "in person"